pkgs/tools/text/frogmouth/default.nix

```diff
@@ -20,6 +20,7 @@ python3.pkgs.buildPythonApplication rec {
   # See <https://github.com/Textualize/frogmouth/pull/59>. So this is a minimal fix.
   postUnpack = ''
     sed -i -e "s,from xdg import,from xdg_base_dirs import," $sourceRoot/frogmouth/data/{config,data_directory}.py
+    sed -i -e 's,\bxdg\b,xdg-base-dirs,g' $sourceRoot/pyproject.toml
```

Comment:
this could be handled by using pythonRemoveDeps
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook